plugins/services/src/js/components/forms/NetworkingFormSection.js

```diff
@@ -526,15 +526,28 @@ class NetworkingFormSection extends mixin(StoreMixin) {
   }
 
   getVirtualNetworks() {
+    const mesosContainer =
+      findNestedPropertyInObject(this.props.data, `container.type`) === "MESOS";
+    // Networks with subnet6 should be disabled when "UCR" container type is selected.
+    const virtualNetworkIsAvailable = (mesosContainer, subnet6) =>
+      !mesosContainer || !subnet6;
+
     return VirtualNetworksStore.getOverlays()
       .mapItems(overlay => {
         const name = overlay.getName();
 
         return {
+          enabled: overlay.info.enabled,
```

Comment:
🎨 You could omit these additional props if you filter the list of overlays first.